CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to this project will be documented in this file.
 
+### [TO BE DEFINED] 2019.04.26
+#### Added
+* Added support for copying Bean into Object populated  with  Builder Pattern
```

Comment:
I think this can be improved, if I were in your shoes, I would fix the following things:

1. Please separate the words with one space only (there are two spaces between `populated` and `with` and between `with` and `Builder`
2. Could you kindly modify the description in order to be coherent with the issue #29? something like: `Added support for the transformation of Java Beans built through Builder` 
Thanks